widgetssdk/src/main/java/com/glia/widgets/core/chathead/domain/IsDisplayChatHeadUseCase.kt

```diff
@@ -23,8 +23,12 @@ internal abstract class IsDisplayChatHeadUseCase(
 ) {
     abstract fun isDisplayBasedOnPermission(): Boolean
 
-    open operator fun invoke(viewName: String?): Boolean {
-        return isBubbleEnabled() && isDisplayBasedOnPermission() && isShowForEngagement(viewName)
+    open operator fun invoke(viewName: String?, internal: Boolean = false): Boolean {
```

Comment:
Could you please help me to understand, where is the value of `USE_OVERLAY` setting taken into account? I see that for `isDisplayApplicationChatHeadUseCase` `internal` is always `true`. I see that if not passed , internal is `false`. But what if permission for showing a bubble outside of the app is granted and `USE_OVERLAY` is `true` - will the bubble be shown outside of the app? I cannot find this case in the code. 🤔

Reply:
Based on my testing, with permission enabled and `USE_OVERLAY` set to false, bubble is still shown inside and outside the app. Which is not expected, as far as I understand, isn't it? As far as I understand, bubble should not be shown if `USE_OVERLAY` is false. Could you please test this case on your own manually?

Reply:
I am guessing you had the overlay permission already set to allowed before changing the use_overlay parameter to false, right?

Reply:
Right.

Reply:
I fixed the issue in the latest fixup

Reply:
The result of the last fix is:
* Bubble is always shown internally
* Bubble is shown outside application with permission and USE_OVERLAY is true
* Even with granted permission, but USE_OVERLAY false the bubble will not be shown outside the application